anago

```diff
@@ -354,8 +354,8 @@ prepare_tree () {
     beta*|official) rev_version_base $label ;;
   esac
 
-  # versionize docs
-  if [[ "$PARENT_BRANCH" == master ]]; then
+  # versionize docs on new branches (from master) only
+  if [[ "$PARENT_BRANCH" == master && "$label" != "alpha" ]]; then
```

Comment:
I don't understand this, but can you just check if you're on the master branch? We should never run versionize docs on the master branch, I'm pretty sure.